pkg/packetbrokeragent/agent.go

```diff
@@ -786,18 +710,14 @@ func (a *Agent) getSubscriptionFilters() []*packetbroker.RoutingFilter {
 
 func (a *Agent) subscribeUplink(ctx context.Context) error {
 	ctx = log.NewContextWithFields(ctx, log.Fields(
-		"namespace", "packetbrokeragent",
 		"home_network_net_id", a.netID,
 		"home_network_tenant_id", a.subscriptionTenantID,
 		"home_network_cluster_id", a.homeNetworkClusterID,
 		"group", a.clusterID,
 	))
 	logger := log.FromContext(ctx)
 
-	conn, err := a.dialContext(ctx, a.dataPlaneAddress,
-		grpc.WithBlock(),
-		grpc.FailOnNonTempDialError(true),
-	)
+	conn, err := a.dialContext(ctx, a.dataPlaneAddress)
```

Comment:
This should likely be reverted as `Subscribe()` relies on the availability of the connection

Reply:
I don't think they are required, the tests use a remote gRPC server and they do work.

Reply:
OK, so just so you know how this works; dialing is now non-blocking. The connection is established in the background. Temporary dialing errors, including TLS handshake issues but there are many more, are not caught. Any usage of the client, fails immediately. If you can cope with those failures because those tasks restart, fine, but configuration errors of PBA are not immediately visible in the logs because the gRPC client is silently reconnecting (with DEBUG level in TTS).

Reply:
Thanks, I understand the reason now :pray:. Reverted in all call sites.